frontend/sphinx/conf.py

```diff
@@ -184,6 +184,13 @@ def get_file_from_conf_ini(path_to_file):
 # The name of the Pygments (syntax highlighting) style to use.
 pygments_style = 'sphinx'
 
+# Figures, tables and code-blocks are automatically numbered if they
+# have a caption.
+numfig = True
```

Comment:
In principle, this addition should be OK, but I'll evaluate the impact on the PDF version of other courses.